### PR TITLE
kodi: rebuild for fmt on {ppc64el,arm64}

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

kodi: rebuild for fmt on {ppc64el,arm64}.

This PR to fix:

```
saki@MagRaspi [ ~ ] $ kodi
/usr/lib/kodi/kodi-x11: error while loading shared libraries: libfmt.so.6: cannot open shared object file: No such file or directory
```

Package(s) Affected
-------------------

kodi: 18.9-1
fmt: 7.0.3-2

Security Update?
----------------

No

Architectural Progress
----------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

**Comment by eatradish** [comment]
FTBFS on Loongson3: http://fars.ee/W44T

TLDR:
```
ERROR: gnutls not found using pkg-config
```

**Comment by eatradish** [comment]
LGTM on arm64

**Review by MingcongBai — APPROVED** [review]
